spec.json

```diff
@@ -1,34 +1,62 @@
 {
   "nio/Blink1": {
-    "version": "1.0.0",
+    "version": "2.0.0",
     "description": "Control a [blink(1) dongle](http://blink1.thingm.com/). Note: The blink1 python module has a dependency on pyusb. On some machines it may be that pyusb from pypi is not a recent enough version. If your block connects to the Blink1 device but is unable to fade the colors, try installing from [source](https://github.com/walac/pyusb). In addition to this, you can also try a [newer version](https://github.com/todbot/blink1) of blink1. This todbot repo is the official repo for thingm, yet it is not the one on pypi.",
     "categories": [
       "Hardware"
     ],
     "properties": {
       "color": {
         "title": "Color",
-        "type": "ObjectType",
+        "type": "StringType",
         "description": "(colors= red|green|blue): RGB color to fade to.",
+        "default": "blue"
+      },
+      "fade_milliseconds": {
+        "title": "Time to fade (ms)",
+        "type": "Type",
+        "description": "Time it takes to fade to new color.",
+        "default": "1000"
+      }
+    },
+    "inputs": {
+      "default": {
+        "description": "Any list of signals."
+      }
+    },
+    "outputs": {},
+    "commands": {}
+  },
+  "nio/Blink1RGB": {
+    "version": "2.0.0",
+    "description": "",
+    "categories": [
+      "Hardware"
+    ],
+    "properties": {
+      "color": {
+        "title": "Color",
+        "type": "ObjectType",
+        "description": "",
```

Comment:
from the old Blink1 block: `"description": "(colors= red|green|blue): RGB color to fade to."`